src/Native/Native-BinPlace.proj

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="4.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <PropertyGroup>
+    <BuildConfigurations>
+      netcoreapp;
+      uap;
+      uapaot;
+      netstandard;
+      netfx;
+    </BuildConfigurations>
+  </PropertyGroup>
+  <Import Project="..\dir.props" />
+  <Target Name="Build" DependsOnTargets="BinPlace" />
+  <Target Name="Clean" />
+  <Target Name="Rebuild" DependsOnTargets="Clean;Build" />
+
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.targets))\dir.targets" />
+
+  <Target Name="GetBinPlaceItems">
+    <ItemGroup>
+      <BinPlaceItem Include="$(NativeBinDir)/*.dll" />
+      <BinPlaceItem Include="$(NativeBinDir)/*.pdb" />
+      <BinPlaceItem Include="$(NativeBinDir)/*.lib" />
```

Comment:
I'm not sure but this list contains all of the file extensions of what we were copying down to the runtime/testhost dirs before, which is how I came up with it. `.lib` is only on Window's clrcompression and `.a` is in non-Windows' System.Natve.a